src/en/readallcomicscom/build.gradle

```diff
@@ -0,0 +1,7 @@
+ext {
+    extName = 'ReadAllComics'
+    extClass = '.ReadAllComics'
+    extVersionCode = 3
```

Comment:
Did you make any changes? If yes, you should bump this. It is difficult to see from the commits if you made any changes.

Reply:
I didn't.